crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -1427,6 +1378,136 @@ unsafe impl SystemParam for SystemChangeTick {
     }
 }
 
+// SAFETY: `init_state` does no access.  The interesting safety checks are on the `SystemParamBuilder`.
```

Comment:
This safety comment isn't clear enough IMO. Make sure to tie back to the safety requirements of `SystemParam` explicitly.